examples/oneapi/cpp/makefile_mac

```diff
@@ -0,0 +1,155 @@
+#===============================================================================
+# Copyright 2014-2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+##  Content:
+##     Intel(R) Data Analytics Acceleration Library examples creation and run
```

Comment:
If we plan to support makefile some time, I think we should clean old wordings in the next PRs (as proposed in #649).